source/common/event/libevent.h

```diff
@@ -31,10 +31,16 @@ namespace Libevent {
  */
 class Global {
 public:
+  static bool initialized() { return is_initialized; }
+
   /**
    * Initialize the library globally.
    */
   static void initialize();
+
+private:
+  // True if initialized() has been called.
+  static bool is_initialized;
```

Comment:
I think is_initialized_ or initialized_ is the correct name, even though it is static:

https://google.github.io/styleguide/cppguide.html#Variable_Names section "Class Data Members", and I didn't see an override in STYLE.md

Reply:
+1